product/opni/components/Clusters.vue

```diff
@@ -32,43 +40,84 @@ export default {
           formatter:     'ListBubbles'
         },
         {
-          name:          'nodes',
-          labelKey:      'tableHeaders.nodes',
-          sort:          ['nodes'],
-          value:         'nodes',
-          formatter:     'List'
+          name:          'capabilities',
+          labelKey:      'tableHeaders.capabilities',
+          sort:          ['capabilities'],
+          value:         'capabilities',
+          formatter:     'ListBubbles'
+        },
+        {
+          name:      'numSeries',
+          labelKey:  'tableHeaders.numSeries',
+          sort:      ['numSeries'],
+          value:     'numSeries',
+          formatter: 'Number'
+        },
+        {
+          name:          'sampleRate',
+          labelKey:      'tableHeaders.sampleRate',
+          sort:          ['sampleRate'],
+          value:         'sampleRate',
+          formatter:     'Number',
+          formatterOpts: { suffix: '/s' }
+        },
+        {
+          name:          'rulesRate',
+          labelKey:      'tableHeaders.rulesRate',
+          sort:          ['rulesRate'],
+          value:         'rulesRate',
+          formatter:     'Number',
+          formatterOpts: { suffix: '/s' }
         },
       ]
     };
   },
 
   created() {
     this.$on('remove', this.onClusterDelete);
+    this.$on('edit', this.openEditDialog);
+    this.$on('copy', this.copyClusterID);
+    this.statsInterval = setInterval(this.loadStats, 10000);
   },
 
   beforeDestroy() {
     this.$off('remove');
+    this.$off('edit');
+    this.$off('copy');
+    if (this.statsInterval) {
+      clearInterval(this.statsInterval);
+    }
   },
 
   methods: {
     onClusterDelete() {
       this.load();
     },
 
-    openCreateDialog(ev) {
-      ev.preventDefault();
-      this.$refs.dialog.open();
+    openEditDialog(cluster) {
+      this.$refs.dialog.open(cluster);
+    },
+
+    copyClusterID(cluster) {
+      this.$copyText(cluster.id);
     },
 
     async load() {
       try {
         this.loading = true;
-        await this.$set(this, 'clusters', await getClusters(this));
+        this.$set(this, 'clusters', await getClusters(this));
       } finally {
         this.loading = false;
       }
+    },
+    async loadStats() {
+      const details = await getClusterStats(this);
+
+      this.clusters.forEach((cluster) => {
+        cluster.stats = details.find(d => d.userID === cluster.id);
```

Comment:
I'd have to look a little closer but I think that this is adding a property to cluster that won't be observable which could cause confusion later if we expect a computed property or watcher to fire if this property were modified. It would be safer to call this.$set(cluster, 'stats', details...) to ensure it's observable.